### PR TITLE
Fix polaris

### PR DESCRIPTION
**Pull request by jdartigalongue**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/830#" title="Author self-approved">jdartigalongue</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/830#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/830#pullrequestreview-286155392" title="Approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/830#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/830#pullrequestreview-286155392" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/830#pullrequestreview-286157586" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: soperator/soperator_resources.go [hunk 1]
```diff
@@ -565,7 +575,7 @@ func (specConfig *SpecConfig) getOperatorRole() *rbacv1.Role {
 			{
 				Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"},
 				APIGroups: []string{"batch", "extensions"},
-				Resources: []string{"jobs", "cronjobs", "ingresses"},
+				Resources: []string{"jobs", "cronjobs", "ingresses", "endpoints", "namespaces"},
```
Comment (yashbhutwala) [inline soperator/soperator_resources.go]: shouldn't matter, but will need to remove create/delete actions from cluster scope resources in namespaced scope roles

**Pull request merged**